### PR TITLE
Update apicatalog.com HTTP API tags

### PR DESCRIPTION
**Pull request by filip26**
added
 * eddsa-2022
 * ecdsa-2019
 * vc2.0

### CODE REVIEW

File: implementations/ApiCatalog.json [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
+    "tags": ["vc-api", "Ed25519Signature2020", "vc2.0", "eddsa-2022", "ecdsa-2019" ]
   }],
   "issuers": [{
     "id": "did:key:z6Mkska8oQD7QQQWxqa7L5ai4mH98HfAdSwomPFYKuqNyE2y",
     "endpoint": "https://vc.apicatalog.com/credentials/issue",
     "options": {
       "type": "Ed25519Signature2020"
     },
-    "tags": ["vc-api", "Ed25519Signature2020"]
+    "tags": ["vc-api", "Ed25519Signature2020", "vc2.0", "eddsa-2022", "ecdsa-2019" ]
```
Comment (JSAssassin) [inline implementations/ApiCatalog.json:16]: @filip26 Currently, the issuer seems to be only issuing `eddsa-2022` type VCs. Should the issuer here be separated into multiple issuers with their `specific tags` and `options.type`? cc: @aljones15

Comment (filip26) [reply implementations/ApiCatalog.json:16]: @JSAssassin what about to pass a requested cryptosuite as a parameter?

Comment (JSAssassin) [reply implementations/ApiCatalog.json:16]: @filip26 I tried passing `options.type = 'ecdsa-2019'` but that didn't work. It's still issuing an VCs with `eddsa-2022`. Not sure what I'm missing.

Comment (filip26) [reply implementations/ApiCatalog.json:16]: yeah, let's me fix it. I was actually waiting to get some request s to see what option is used to request a cryptostuite.  Let's use `options.type` . Possible values:

* Ed25519Signature2020
* eddsa-2022
* ecdsa-2019

The current implementation defaults to eddsa-2022. Thank you!

Comment (JSAssassin) [reply implementations/ApiCatalog.json:16]: @filip26 sounds good! Thanks.

Comment (aljones15) [reply implementations/ApiCatalog.json:16]: @filip26 the VC API no longer seems to have that option just so you know.

Comment (filip26) [reply implementations/ApiCatalog.json:16]: @aljones15 ok, so how do we ask the issuer for specific cryptosuite?

Comment (JSAssassin) [reply implementations/ApiCatalog.json:16]: @aljones15 I'm not entirely sure which VC API you are referring to, but as long as Filip sets a default for `options.type` for the implementation on his end and the implementation with `vc-api` tag is able to issue a signed VC, it should be good, no? The old VC API tests AFAIK do not specifically test the proof type. This means any issuer implementation can be used for that test suite as long as it issues a signed VC.

**Pull request merged**